frontend/device/kindle/device.lua

```diff
@@ -46,6 +46,9 @@ function Kindle:usbPlugIn()
 end
 
 function Kindle:intoScreenSaver()
+    if not require("ui/screensaver"):isKindleSpecialOffers() and require("ui/screensaver").isUsingBookCover then
```

Comment:
if you are doing multiple require here, might as well require once as save it to a local var for reuse:

```lua
local Screensaver = require("ui/screensaver")
if not self:hasSpecialOffers() and not Screensaver:isUsingBookCover() then
    Screensaver:show("suspend")
end
```

One tip I would like to point out is the coding style convention: we use `under_score_name` for variables and `camelCaseName` for functions. With that in mind, it's easy to spot that `require("ui/screensaver").isUsingBookCover` is a typo ;)